build-cs/composer.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Will Dependabot be aware of this package manifest?

Reply:
No, it will need to be added to the Dependabot config. But this project does not appear to be using Dependabot.

Reply:
Not explicitly, but Dependabot still [sending security updates](https://github.com/10up/plugin-check/pulls?q=is%3Apr+is%3Aclosed+label%3Adependencies).

Reply:
Opened #168